applications/nrf_desktop/Kconfig.defaults

```diff
@@ -24,4 +24,10 @@ choice SETTINGS_BACKEND
 	default SETTINGS_NVS
 endchoice
 
+config SETTINGS_DYNAMIC_HANDLERS
+	default n
```

Comment:
`default n` is not needed as n is default option if no default is given

Reply:
It's needed here, because we overlay default of an existing Kconfig option, not introducing a new one.